<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Cars</class>
 <widget class="QWidget" name="Cars">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1179</width>
    <height>642</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>20</y>
     <width>781</width>
     <height>601</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="addCar">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>500</y>
     <width>171</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Добавить автомобиль</string>
   </property>
  </widget>
  <widget class="QPushButton" name="delCar">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>530</y>
     <width>171</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Удалить автомобиль</string>
   </property>
  </widget>
  <widget class="QPushButton" name="checkAuto">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>560</y>
     <width>171</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>все авто</string>
   </property>
  </widget>
  <widget class="QPushButton" name="searchByNum">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>500</y>
     <width>191</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Найти авто по номеру</string>
   </property>
  </widget>
  <widget class="QPushButton" name="checkByOptions">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>530</y>
     <width>191</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Найти авто по критериям</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="num">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>50</y>
     <width>113</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="year">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>120</y>
     <width>113</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="carname">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>260</y>
     <width>311</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>20</y>
     <width>63</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Номер</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>90</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Год выпуска</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>230</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Название авто</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>20</y>
     <width>63</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Пробег</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="probeg">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>50</y>
     <width>113</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QCheckBox" name="bitai">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>120</y>
     <width>93</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>Битая</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>290</y>
     <width>63</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Опции</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="sits">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>320</y>
     <width>161</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>Подогрев сидений</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="rule">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>360</y>
     <width>141</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>Подогрев руля</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="antirain">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>400</y>
     <width>141</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>Антидождь</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="top">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>440</y>
     <width>181</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>Панорамная крыша</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>190</y>
     <width>82</width>
     <height>28</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>A</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>B</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>C</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>D</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>E</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>F</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>160</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Класс авто</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
